packages/fxa-payments-server/src/components/PlanDetails/index.stories.tsx

```diff
@@ -104,14 +104,14 @@ export const Default = storyWithProps({
   showExpandButton: false,
 });
 
-export const DefaulWithInclusiveTax = storyWithProps({
+export const DefaultWithInclusiveTax = storyWithProps({
   selectedPlan,
   isMobile: false,
   showExpandButton: false,
   invoicePreview: invoicePreviewInclusiveTax,
 });
 
-export const DefaulWithExclusiveTax = storyWithProps({
+export const DefaultWithExclusiveTax = storyWithProps({
```

Comment:
Thank you for correcting. 🙂

I am not seeing a difference in both stories. I keep getting undefined for `invoicePreview`. Is there something I am missing?

Reply:
I was not seeing a difference in the stories either - I'll ping Ivo, though further updates could fall out of scope for this ticket.

Reply:
Aligned. After discussing in Slack I took a look and had the same problem in main. It appears that `!taxAmount` is always resulting in true, resulting in all tax details getting omitted. This requires further attention that is out of scope for this ticket. I agree that we should file a new ticket to address there (that would also look at routes that include `PlanDetails`).

Reply:
Thanks for looking into this, @sardesam ! I've opened a [ticket](https://mozilla-hub.atlassian.net/browse/FXA-6320). I'll go ahead and squash commits, please let me know if there are any further changes required for approval. Thanks again!